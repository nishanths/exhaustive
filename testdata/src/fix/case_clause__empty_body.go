package fix

import (
	bar "general/y"
)

func _b() {
	var phy bar.Phylum
	switch phy { // want "^missing cases in switch of type bar.Phylum: Chordata, Echinodermata, Mollusca$"
	}
}
