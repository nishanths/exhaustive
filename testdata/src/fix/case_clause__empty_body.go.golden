package fix

import (
	"fmt"
	bar "general/y"
)

func _b() {
	var phy bar.Phylum
	switch phy { // want "^missing cases in switch of type bar.Phylum: Chordata, Echinodermata, Mollusca$"
	case bar.Chordata, bar.Echinodermata, bar.Mollusca:
		panic(fmt.Sprintf("unhandled value: %v", phy))
	}
}
